code/game/objects/items/devices/litanybook.dm

```diff
@@ -42,4 +42,15 @@
 
 /obj/item/device/litanybook/gadpathur/Initialize()
 	. = ..()
-	randomquip = file2list("ingame_manuals/gadpathur.txt")
\ No newline at end of file
+	randomquip = file2list("ingame_manuals/gadpathur.txt")
+
+/obj/item/device/litanybook/biesel
```

Comment:
If you are going to borrow litany book code, make the litany book mechanical functions into a generic class called `verse_book` or something, then make the litany book and this constitution into subtypes.